OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -72,6 +72,9 @@ public class AircraftInfo : ITraitInfo, IPositionableInfo, IFacingInfo, IMoveInf
 		[Desc("Can the actor hover in place mid-air? If not, then the actor will have to remain in motion (circle around).")]
 		public readonly bool CanHover = false;
 
+		[Desc("Can the actor change direction without having to change facing? If not, the actor will have to fly a curve to change direction.")]
+		public readonly bool CanChangeDirectionWithoutTurning = false;
```

Comment:
We're growing quite a family of flags now. What if we pushed these into a `FlightDynamics` enum with flags for `VTOL`, `Hovers`, `Slides`, `LandWhenIdle`, and so on. The `ActionOnIdle` cases can then hopefully be included here too.

Reply:
Sounds like a good idea. That will certainly take longer to get reviewed and merged though, so I'll split that to its own PR and leave only the AttackType commit in this one.